api/client/src/main/java/org/projectnessie/client/api/NessieApiV2.java

```diff
@@ -32,6 +33,8 @@
  */
 public interface NessieApiV2 extends NessieApiV1 {
 
+  String toPathString(ContentKey key);
```

Comment:
WDYT about moving this declaration to `HttpApiV2`? Technically "path strings" are relevant at the HTTP level.

Callers of this method are also all in the `rest/v2` package.